components/compositing/compositor.rs

```diff
@@ -862,20 +867,69 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
         }
     }
 
+    /// Create a WebRender display list for the root pipeline in the given transaction.
+    /// This display list contains a full-screen iframe with the root content pipeline
+    /// with any pinch zoom transform applied to it.
+    fn set_root_display_list(&self, transaction: &mut Transaction) {
+        let root_content_pipeline = match self.root_content_pipeline.id {
+            Some(id) => id,
+            None => return,
+        };
+
+        let root_pipeline = WebRenderPipelineId(0, 0);
```

Comment:
Is the `(0, 0)` pipelineId guaranteed to unused by normal pipelines?

Reply:
Yes, though it is true that is also what WebRender uses for it's `dummy` pipeline. I've changed this to `(0, 1)` and added a comment explaining why it shouldn't conflict.